examples/huggingface-transformers/README.md

```diff
@@ -14,137 +14,93 @@ See the License for the specific language governing permissions and
+```python
+from deepsparse.transformers import pipeline
 
-The app exposes HTTP endpoints at:
-- `/info` to get information about the compiled model
-- `/predict` to send inputs to the model and receive a response.
-    The number of inputs should match the compiled model's batch size.
+qa_pipeline = pipeline("question-answering")
 
-For a full list of options, run `python server.py -h`.
+inference = qa_pipeline(question="What's my name?", context="My name is Snorlax")
+```
+**more tasks coming soon...😇**
+## Benchmarking & the DeepSparse Server
 
-Currently, the server uses DeepSparse-HuggingFace pipeline integration 
-for end-to-end prediction.  
+### Benchmarking ONNX Models 📜
 
-### Client
+💾 [List of the the Hugging Face SparseZoo Models](https://sparsezoo.neuralmagic.com/?repo=huggingface&page=1)
 
-`client.py` provides a callable `PipelineClient` object to make requests to the 
-server easy.
-The file is self-documented.  See example usage below:
+Want to find out how fast our sparse Hugging Face ONNX models perform inference? You can quickly do benchmarking tests with CLI commands; you only need to provide the model path of a SparseZoo ONNX model or your own local ONNX model to get started:
 
-```python
-from client import PipelineClient
-remote_model = PipelineClient(address='0.0.0.0', port='5543')
-model_outputs = remote_model(question="What's my name?", context="My name is Snorlax")
+```bash
+deepsparse.benchmark zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned-aggressive_98
 ```
 
-### SparseZoo Stubs
-[SparseZoo](http://sparsezoo.neuralmagic.com/) is a constantly-growing repository of sparsified (pruned and 
-pruned-quantized) models with matching sparsification recipes for neural networks. It simplifies and accelerates your time-to-value in building performant deep learning models with a collection of inference-optimized models and recipes to prototype from.
-
-Available via API and hosted in the cloud, the [SparseZoo](http://sparsezoo.neuralmagic.com/) contains both 
-baseline and models sparsified to different degrees of inference performance vs. baseline loss recovery. Recipe-driven approaches built around sparsification algorithms allow you to take the models as given, transfer-learn from the models onto private datasets, or transfer the recipes to your architectures.
+For a more in-depth discussion on benchmarking, check out the [Benchmarking tutorial](https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/benchmark_model)!
 
-Each model in the SparseZoo has a specific stub that identifies it. The stubs are made up of the following structure:
+### DeepSparse Server 🔌
 
-`DOMAIN/SUB_DOMAIN/ARCHITECTURE{-SUB_ARCHITECTURE}/FRAMEWORK/REPO/DATASET{-TRAINING_SCHEME}/SPARSE_NAME-SPARSE_CATEGORY-{SPARSE_TARGET}`
+The DeepSparse inference server allows you to serve ONNX models and pipelines in HTTP. 
```

Comment:
Make sure to mention you need `pip install "deepsparse[server]"`, thanks @dbarbuzzi